src/System.Private.CoreLib/src/Microsoft/Win32/SafeHandles/SafeRegistryHandle.Windows.cs

```diff
@@ -1,19 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Runtime.InteropServices;
-
-namespace Microsoft.Win32.SafeHandles
-{
-#if REGISTRY_ASSEMBLY
-    public
-#else
-    internal
-#endif
-    sealed partial class SafeRegistryHandle : SafeHandle
-    {
-        protected override bool ReleaseHandle() =>
```

Comment:
I have missed that you have merged SafeRegistryHandle.Windows.cs and SafeRegistryHandle.cs. It is ok for CoreRT/CoreCLR, but it may be a problem for reusing the same file CoreFX - CoreFX has these split to support (unfinished) implementation of filesystem-based registry mock.